boot/zephyr/Kconfig

```diff
@@ -66,6 +66,17 @@ config BOOT_SIGNATURE_TYPE_RSA
 	select BOOT_USE_MBEDTLS
 	select MBEDTLS
 
+if BOOT_SIGNATURE_TYPE_RSA
+choice
```

Comment:
You might want to consider using a `range` here (between the 2048 and 3072 values), and just documenting that only those two values are valid. Not sure though; would like to know what people think.

Reply:
The problem is that `imgtool` is still only able to generate RSA 2048 and 3072 at the moment, and I am not sure we should allow for free choice of number of bits (even between a range).

Reply:
Totally fair -- I'm thinking that the value could be a `_LEN` here, which could e.g. be checked in the header file (`#if CONFIG_BOOT_SIGNATURE_TYPE_RSA_LEN != 2048 && .... != 3072 / #error / #endif`).

Would make the zephyr config easier to change later -- just one option, which is the number of bits, but without sacrificing the restriction that only some are valid.

Reply:
Ok, updated to use a `range`.

Reply:
Thanks!